bullet_train-fields/package.json

```diff
@@ -48,6 +48,7 @@
     "emoji-mart": "^5.1.0",
     "microbundle": "^0.13.0",
     "moment-timezone": "^0.5.43",
+    "monaco-editor": "^0.52.2",
```

Comment:
Since we're not actually using this package directly I think it would be better to remove this, so that we don't end up forcibly including this as part of an application JS bundle. Is that reasonable/possible?

Reply:
Because `code_editor_controller.js` is only extracting the current version number (for fetching a precise CDNed version), nothing else from the package is included in the application.js output. You can verify this via `assets:precompile`.

We should keep this here to benefit from dependencies auto-updates via CI.

Reply:
So, if we're gonna have it in `package.json` why load it from CDN? Seems like an unusual pattern that I haven't seen before and don't really understand.

Reply:
But you're asking why load it from CDN. Because this package is heavy, and is best loaded in pieces: the theme file you need, the code language you need.

Reply:
OK, I think that makes sense.